before_install.sh

```diff
@@ -59,8 +59,8 @@ fi)
 
 #TODO: work out how to avoid the need for this
 chmod u+x packages/FSharp.Compiler.Tools.4.1.5/tools/fsi.exe 
```

Comment:
NuGet.exe has the option to exclude the version number in path. See [cli install](https://docs.microsoft.com/en-us/nuget/tools/nuget-exe-cli-reference#install).
The caveat here is that the VS client doesn't seem to support this.